### DISCUSSION TITLE
Still here

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1161] [Team Posts]
S.N.A.F.U. Thanks for 'minding the store'.
Hope to have a commit soon, a **very** difficult fix I've been **quietly** working on for weeks,
 along with a couple of dire family emergencies.
And now this...
Hope you are all well.
Tim.

**Comment by spamatica** [comment]
Hey Tim!

Yeah, family emergencies tend to occur once in a while... hope it works out.

Things are slowly getting kind of crazy over here, not that we are that far
gone.. but it's just corona corona corona 24/7 on the news.

Good to hear from you! Looking forward to your next commit.
/Robert

Den ons 18 mars 2020 kl 17:26 skrev Tim <notifications@github.com>:
